en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -0,0 +1,114 @@
+# Kafka Connector Example
+
+Given below is a sample scenario that demonstrates how to send messages to a Kafka broker via Kafka topics.
+
+## What you'll build
+The publishMessages operation allows you to publish messages to the Kafka brokers via Kafka topics.
```

Comment:
Shall we merge this sentence with the above? They are too similar.

Reply:
Merged.